vispy/scene/cameras/cameras.py

```diff
@@ -630,7 +631,7 @@ def viewbox_mouse_event(self, event):
         BaseCamera.viewbox_mouse_event(self, event)
         
         if event.type == 'mouse_wheel':
-            center = self._scene_transform.imap(event.pos)
+            center = np.array(self._scene_transform.imap(event.pos))[0]
```

Comment:
In addition to changing what `*` means from broadcast-multiply to dot (under most circumstantces), I found another thing we must keep in mind when using `matrix` instead of `ndarray`. If you take a dot product between a matrix and a 1D vector, you get a matrix (2D) back:
```
>>> import numpy as np
>>> np.dot(np.matrix(np.ones((2, 2))), np.ones(2))
matrix([[ 2.,  2.]])
```
We'll have to keep exceptions like this in mind now that we have adopted use of the `matrix` class in a subset of our operations.